arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/RaiseAccumulate.kt

```diff
@@ -709,7 +770,7 @@ public open class RaiseAccumulate<Error>(
 
   @RaiseDSL
   @JvmName("_mapOrAccumulate")
-  public inline fun <A, B> mapOrAccumulate(
+  public fun <A, B> mapOrAccumulate(
```

Comment:
Without `inline` we don't support `suspend` here, and thus break "traverse" for suspend.

Reply:
I don't know why I've removed it, but it's now back